pytrends/request.py

```diff
@@ -285,7 +283,7 @@ def interest_by_region(self, resolution='COUNTRY', inc_low_vol=False,
         # rename the column with the search keyword
         df = df[['geoName', 'geoCode', 'value']].set_index(
```

Comment:
For resolution 'CITY' the column 'geoCode' is not delivered. Hence running here into `KeyError: "['geoCode'] not in index"`